File: Andoid/Intent/phone_cliant/app/src/main/java/com/example/sample/phone_cliant/MainActivity.java
package com.example.sample.phone_cliant;

import android.content.Intent;
import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button button = findViewById(R.id.test);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Uri uri = Uri.parse("tel:1111111111");
                Intent i = new Intent(Intent.ACTION_DIAL,uri);
                startActivity(i);
            }
        });
    }
}

File: Andoid/kotlin/MyApplication/app/src/main/java/com/example/kotlin/myapplication/MainActivity.kt
package com.example.kotlin.myapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    private external fun StringForJNI(): String;
    private external fun SumForJNI(i : Int, j : Int): Int;


    companion object {
        init {
            System.loadLibrary("native-lib")
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val str = "1 + 2 = " + SumForJNI(1, 2)
        val view = findViewById<TextView>(R.id.text)
        view.setText(str)
    }
}
